py/jupyterlite/src/jupyterlite/__init__.py

```diff
@@ -1,4 +1,7 @@
-"""tools for building JupyterLite sites
+"""re-export the core constants and addons from the core package.
 """
 
 __version__ = "0.1.0b18"
+
+from jupyterlite_core.addons.base import *  # noqa
```

Comment:
welp, as we've had to shim _a couple things_ around here... probably need individual files to properly emulate the modules. star imports are fine. probably _do_ want a warning.